### DISCUSSION TITLE
Unable to access expose.dev

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [exposedev/expose#311] [General]
@mpociot I am unable to access https://expose.dev. I get the access denied error

![CleanShot 2022-03-17 at 16 21 01](https://user-images.githubusercontent.com/1639302/158793811-4b0330be-fe7a-4531-9559-01cb2e905915.png)

**Comment by abishekrsrikaanth** [comment]
Looks like the domain is blocked for all of India. I tried multiple VPN servers here in India and I still can't access it. But if I switch to countries other than India, I can access it.

**Reply by tanhatariq** [reply]
you are probably right @abishekrsrikaanth, even i have tried all the possible ways to connect to expose.dev from India, VPN, Apple Private Relay etc. Even if we manage to open website, sign up and generate token through VPN, the same will not be identified by HERD from Indian IP address. Herd will keep saying that your token is invalid.

**Reply by abishekrsrikaanth** [reply]
No updates, been requesting to open this up on cloudflare, India is restricted on their cloudflare firewall for some reason. India probably has major chunk of developers and they probably don't want spending money on the bandwidth from here, thats the only reason that I can think of. Never get any answers for this issue that i have raised at multiple places including twitter multiple times directly to @mpociot.

**Comment by LemonPie22** [comment]
I have a similar problem accessing the site from the UK

**Comment by ranjan3nov** [comment]
I have similar problem is there any solution for it
